ImperatorToCK3/Data_Files/configurables/culture_map.txt

```diff
@@ -543,7 +568,11 @@ link = { ck3 = old_egyptian @ir_egyptian_cultures }
 link = { ck3 = baranis ir = baranis } # https://steamcommunity.com/sharedfiles/filedetails/?id=3098496649
 link = { ck3 = butr ir = butr } # https://steamcommunity.com/sharedfiles/filedetails/?id=3098496649
 # TFE
+link = { ck3 = garamantian ir = nasamonian ir = macaean ir = psyllic ir = garamantic ir = libyan ir = berber ir = phazani ir = lotophagoi ir = gyzantian }
```

Comment:
Sort of? I put Berber there because it's in the Fezzani group which seem to completely line up with the Garamantian culture in TFE while every other Berber culture correlates to the Numidian culture group. That said, I get why the Berber culture specifically would be applied to any split since it's so broad and shares a name with the CK3 group so it could be moved bellow if you want.

Reply:
I'm fine with the current state of the PR, I checked in Invictus files and the `berber` cultures doesn't exist on the map anyway.